tools/__init__.py

```diff
@@ -19,14 +21,19 @@
 
 # Put all tool files in __all__
 # allows "from tools import *" to import all tooles for testtools
-__all__ = [filename[:-3]  # Remove .py
-           for filename in os.listdir(os.path.dirname(__file__))  # tools directory
-           if filename.endswith('.py') and filename != '__init__.py' and filename not in [  # Add any files to exclude here:
-               # e.g. 'sometool.py',
-           ]]
+__all__ = sorted(
+    [
+        filename[:-3]    # Remove .py
+        for filename in os.listdir(os.path.dirname(__file__))    # tools directory
+        if filename.endswith(
+            '.py') and filename != '__init__.py' and filename not in [    # Add any files to exclude here:
+    # e.g. 'sometool.py',
+            ]
+    ]
+)
 installed_tools = {}
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
```

Comment:
The all-caps version was due to pylint asserting that anything in the global scope should be in all caps. Since the logger object is not a constant, the `LOG` should probably be `_log` per the PEP8 [guideline](https://www.python.org/dev/peps/pep-0008/#global-variable-names) for non-public module globals. I'll make that change in a separate PR, and revert the names in this PR to be all caps for consistency.